connexion/apps/flask_app.py

```diff
@@ -101,7 +103,8 @@ def run(self, port=None, server=None, debug=None, host=None, **options):  # prag
 
         logger.debug('Starting %s HTTP server..', self.server, extra=vars(self))
         if self.server == 'flask':
-            self.app.run(self.host, port=self.port, debug=self.debug, **options)
+            self.app.run(self.host, port=self.port, debug=self.debug,
+                    extra_files=self.extra_files, **options)
```

Comment:
The checks are failing because this line is not correctly indented. It should be aligned with the `self.host` in the line above (add 5 spaces I think).